stdlib/public/Concurrency/Actor.swift

```diff
@@ -15,7 +15,7 @@ import Swift
 
 /// Common protocol to which all actors conform.
 ///
-/// The \c Actor protocol generalizes over all actor types. Actor types
+/// The Actor protocol generalizes over all actor types. Actor types
```

Comment:
```suggestion
/// The `Actor` protocol generalizes over all actor types. Actor types
```

Reply:
Thought that was in an abstract, but you're right, it's a discussion, so it can take code voice.